src/app/components/ag-grid/index.html

```diff
@@ -0,0 +1,145 @@
+<sky-tabbed-demo-page
+  pageTitle="ag-Grid"
+  navTitle="ag-Grid"
+  skyuxModule="SkyAgGridModule"
+  packageName="@skyux/ag-grid"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The SKY UX ag-Grid module provides styles, a service for default gridOptions, and SKY UX components for cell rendering and editing in AG grid. This page primarily describes the design use cases for grids and how to use the SKY UX ag-Grid service. If you have never used ag-Grid before, <a href="https://www.ag-grid.com/angular-getting-started/">the Angular getting started</a> guide covers the basics. For more in depth information about ag-Grid and its capabilities, please reference <a href="https://www.ag-grid.com/documentation-main/documentation.php">the ag-Grid documentation.</a>
```

Comment:
Should this be `... the Angular *Grid* getting started guide covers the basics ...`?

Reply:
I don't think so per se, ag-Grid actually stands for agnostic grid, not angular grid. So the Angular getting started guide is just the angular version of getting started with agnostic grid.

Reply:
I don't think so per se, ag-Grid actually stands for agnostic grid, not angular grid. So the Angular getting started guide is just the angular version of getting started with agnostic grid.